src/main/java/com/fundynamic/d2tm/Game.java

```diff
@@ -35,6 +35,8 @@ public void initStatesList(GameContainer container) throws SlickException {
                 TILE_HEIGHT
         );
 
+        container.setVSync(true);
```

Comment:
I know we spoke about this, but I removed this on purpose yesterday. Since the lagginess is fixed on your computer I think it's nice to allow a high framerate.

On the other hand, we'll probably make it configurable later on :wink: